examples/prebid-cstg/html/index.html

```diff
@@ -0,0 +1,192 @@
+      });
+
+      var pbjs = pbjs || {};
+      pbjs.que = pbjs.que || [];
+      pbjs.que.push(function () {
+        $(document).ready(onDocumentReady);
+      });
+
+      function sendAdserverRequest() {
+        if (pbjs.adserverRequestSent) return;
+        pbjs.adserverRequestSent = true;
+        googletag.cmd.push(function () {
+          pbjs.que.push(function () {
+            pbjs.setTargetingForGPTAsync();
+            googletag.pubads().refresh();
+          });
+        });
+      }
+
+      setTimeout(function () {
+        sendAdserverRequest();
+      }, FAILSAFE_TIMEOUT);
+    </script>
+    <script>
+      (function () {
+        var gads = document.createElement('script');
+        gads.async = true;
+        gads.type = 'text/javascript';
+        gads.src = 'https://securepubads.g.doubleclick.net/tag/js/gpt.js';
+        var node = document.getElementsByTagName('script')[0];
+        node.parentNode.insertBefore(gads, node);
+      })();
+    </script>
+    <script>
+      googletag.cmd.push(function () {
+        googletag
+          .defineSlot(
+            '/112115922/FL_PB_MedRect',
+            [
+              [300, 250],
+              [300, 600],
+              [728, 90],
+            ],
+            'test-div'
+          )
+          .addService(googletag.pubads());
+        googletag.pubads().enableSingleRequest();
+        googletag.enableServices();
+      });
+    </script>
+    <script>
+      function setUid2Config(email) {
+        const cstgParams = {
+          uid2ApiBase: '{{ UID2_BASE_URL }}',
+          subscriptionId: '{{ SUBSCRIPTION_ID }}',
+          serverPublicKey: '{{ SERVER_PUBLIC_KEY }}',
+        };
+        if (email) cstgParams.email = email;
+        pbjs.setConfig({
+          debug: true,
+          userSync: {
+            userIds: [
+              {
+                name: 'uid2',
+                params: cstgParams,
+              },
+            ],
+            syncDelay: 5000,
+            auctionDelay: 1000,
+          },
+        });
+      }
+
+      async function updateGuiElements() {
+        const uid2  = pbjs.getUserIds().uid2;
+        $('#targeted_advertising_ready').text(uid2 ? 'yes' : 'no');
+        $('#advertising_token').text(uid2 ? String(uid2.id) : '');
+        if (!uid2) {
+          $('#login_form').show();
+          $('#clear_storage_form').hide();
+        } else {
+          $('#login_form').hide();
+          $('#clear_storage_form').show();
+        }
+      }
+
+      async function handleLogin() {
+        const email = $('#email').val();
+          setUid2Config(email);
+          pbjs.refreshUserIds().then(() => {
+            updateGuiElements();
+          });
+      }
+
+      function handleClearStorage() {
+        localStorage.removeItem('__uid2_advertising_token');
+        location.reload();
+      }
+
+      function onDocumentReady() {
+        $('#login').click(handleLogin);
+        $('#clear_storage').click(handleClearStorage);
```

Comment:
Why do we clear storage? It could be good to demonstrate how the ad token is persisted across visits to the site